spring4/src/main/java/feign/spring/SpringContract.java

```diff
@@ -51,6 +44,42 @@ public SpringContract() {
         data.template().method(requestMapping.method()[0].name());
     });
 
+
+    registerMethodAnnotation(GetMapping.class, (postMapping, data) -> {
```

Comment:
Can you rename all the `postMapping` to `mapping`?

Reply:
sure, missed this one ;)